styles/landing-page.css

```diff
@@ -0,0 +1,131 @@
+:root{
+    --primary-color: #DAA06D;
+}
+
+*{
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+body{
+    margin: 0;
+    padding: 0;
+}
+.main-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.nav-container{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: white;
+}
+.doc-container{
+    display: flex;
+}
+.title{
+    font-size: 2.5rem;
+    margin: 1.5rem;
+    margin-left: 2rem;
+}
+.menu-items ul{
+    margin: 0;
+    padding: 0;
+    display: flex;
+}
+.menu-items li{
+    list-style: none;
+}
+
+.menu-items li a {
```

Comment:
You can create a class by the name as .menu-items-links and pass these styles to it !!So that you can use this class to any of the anchor tags and not just the ones that have these only .menu-items li a{}